stable/aws-vpc-cni/README.md

```diff
@@ -61,3 +56,22 @@ Specify each parameter using the `--set key=value[,key=value]` argument to `helm
 ```shell
 $ helm install --name aws-vpc-cni --namespace kube-system eks/aws-vpc-cni --values values.yaml
 ```
+
+## Adopting the existing aws-node resources in an EKS cluster
+
+If you do not want to delete the existing aws-node resources in your cluster that run the aws-vpc-cni and then install this helm chart, you can adopt the resources into a release instead. This process is highlighted in this [PR comment](https://github.com/aws/eks-charts/issues/57#issuecomment-628403245). Once you have annotated and labeled all the resources this chart specifies, enable the `originalMatchLabels` flag on the helm release and run an update. If you have been careful this should not diff and leave all the resources unmodified and now under management of helm.
+
+Here is an example script to modify the existing resources:
+WARNING: Substitute YOUR_HELM_RELEASE_NAME_HERE with the name of your helm release.
+```
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+for kind in ds clusterRole clusterRoleBinding serviceAccount; do
+  echo "setting annotations and labels on $kind"
```

Comment:
Nit-pick: `$kind/aws-node` so that just the text output is sufficient to know what things were changed. Also matters because the CRD is not named aws-node, so it would be slightly confusing.